test/future_then_tests.cpp

```diff
@@ -352,7 +352,7 @@ BOOST_AUTO_TEST_CASE(reduction_future_move_only_to_void) {
               }).then([& _result = result](auto&& x) {
             return async(
                 default_executor, [&_result](auto&& x) { _result = std::move(x); },
-                forward<move_only>(x));
+                std::forward<move_only>(x));
```

Comment:
Did you consider putting all these `std::` additions into a separate PR?

Reply:
Done. #478.